WooCommerce/src/main/kotlin/com/woocommerce/android/model/ProductAddon.kt

```diff
@@ -27,7 +27,19 @@ data class ProductAddon(
     val type: AddOnType?,
     val display: AddOnDisplay?,
     val options: List<ProductAddonOption>
```

Comment:
💭 If I understood your intentions correctly, we'd like always take `price` through `priceSafeOptionList` right? WDYT about making `price` and `options` private fields?

```suggestion
    private val price: String,
    (...)
    private val options: List<ProductAddonOption>
```

Reply:
Makes perfect sense, I did some overall refactoring on top of your suggestion and submitted it in two commits: https://github.com/woocommerce/woocommerce-android/pull/4491/commits/69c537fe63216158902f1ac9e44c3bc294121ccf and https://github.com/woocommerce/woocommerce-android/pull/4491/commits/0106d90d073c0d61a14bd8dee3043cde64b1aab4

Let me know what you think. Thanks!

Reply:
I love `rawOptions` vs `options`, thanks!